src/main/java/org/wso2/carbon/inbound/poll/dbeventlistener/DBEventPollingConsumer.java

```diff
@@ -30,14 +30,7 @@
 import org.wso2.carbon.context.PrivilegedCarbonContext;
 import org.wso2.carbon.inbound.endpoint.protocol.generic.GenericPollingConsumer;
 
-import java.sql.PreparedStatement;
-import java.sql.SQLException;
-import java.sql.Connection;
-import java.sql.ResultSet;
-import java.sql.ResultSetMetaData;
-import java.sql.Statement;
-import java.sql.DriverManager;
-import java.sql.Types;
+import java.sql.*;
```

Comment:
Shall we add only the necessary imports instead of adding all?

Reply:
fix with commit 77dba88e588a7ed6959f3ce479e96a3abb87ca38